java/sidecar-common/src/main/java/com/instaclustr/sidecar/operations/OperationsResource.java

```diff
@@ -46,6 +46,12 @@ public Operation getOperationById(@NotNull @PathParam("id") final UUID id) {
         return operationsService.operation(id).orElseThrow(NotFoundException::new);
     }
 
+    @POST
+    @Path("filter")
```

Comment:
Typically filtering is done via query parameters, rather than a request body.

e.g., `GET /operations?type=backup;type=cleanup`